staging/src/k8s.io/kubectl/pkg/cmd/diff/diff.go

```diff
@@ -56,7 +56,13 @@ var (
 
 		KUBECTL_EXTERNAL_DIFF environment variable can be used to select your own
 		diff command. By default, the "diff" command available in your path will be
-		run with "-u" (unified diff) and "-N" (treat absent files as empty) options.`))
+		run with "-u" (unified diff) and "-N" (treat absent files as empty) options.
```

Comment:
Actually, can you use the same format as above? I think it's clearer that the long sentence. Thanks!

Reply:
Wait no this part is good, you can keep it as it is.